lib/theme_check/checks/missing_required_template_files.rb

```diff
@@ -3,11 +3,11 @@
 module ThemeCheck
   # Reports missing shopify required theme files
   # required templates: https://shopify.dev/tutorials/review-theme-store-requirements-files
-
+  # https://shopify.dev/docs/themes/theme-templates
```

Comment:
Is this needed?

Reply:
I moved it from below and also in the documentation file. Can definitely remove.